### PR TITLE
Fixed docker-compose for initial build

### PR DESCRIPTION
**Pull request by ArtemBernatskyy**
Fixed issue #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+### FAQ
+
+змінити файл web/src/App.js , web/src/index.css
```
Comment (p1v2) [inline README.md:3]: @ArtemBernatskyy , your Readme.md file doesn't look fine. It is written in Ukrainian language, but English language is preferable. Also it does not contain instructions about how to setup and run the project.

File: client/web/Dockerfile [hunk 2]
```diff
@@ -2,15 +2,13 @@ FROM node:8.12.0
 
 EXPOSE 3000
 
-RUN mkdir /app
-WORKDIR /app
+RUN mkdir /my-app
```
Comment (p1v2) [inline client/web/Dockerfile:5]: `/client` directory name will be more fine here

File: client/web/src/App.js [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
 class App extends Component {
   render() {
     return (
       <div className="App">
         <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
+          <img src={image} className="App-logo" alt="logo" />
           <p>
-            Edit <code>src/App.js</code> and save to reload.
+            Ok, <code>src/App.js</code> here we have Ivan project
```
Comment (p1v2) [inline client/web/src/App.js:12]: That is not only the Ivan's project 😉

File: server/myplay/bin/uwsgi.ini [hunk 4]
```diff
@@ -6,3 +6,4 @@ module = myplay.wsgi:application
 socket = 0.0.0.0:8000
 
 buffer-size = 32768
+workers = 0
```
Comment (p1v2) [inline server/myplay/bin/uwsgi.ini:9]: Just lol

File: server/myplay/core/models.py [hunk 5]
```diff
@@ -1,3 +1,23 @@
 from django.db import models
 
 # Create your models here.
+
+
+# from django.conf import settings
```
Comment (p1v2) [inline server/myplay/core/models.py:6]: Either uncomment and make sure it is working or delete this.

**Review by p1v2 — CHANGES_REQUESTED** [review]
Please see my comments below

File: README.md (hunk 1 above)
Comment (ArtemBernatskyy) [reply README.md:3]: :)))))) I've put it by mistake )))

File: server/myplay/bin/uwsgi.ini (hunk 4 above)
Comment (ArtemBernatskyy) [reply server/myplay/bin/uwsgi.ini:9]: lol

**Pull request closed without merging**